lms/djangoapps/discussion/urls.py

```diff
@@ -1,21 +1,22 @@
 """
 Forum urls for the django_comment_client.
 """
-from django.conf.urls import patterns, url
+from django.conf.urls import url
 
 from .views import DiscussionBoardFragmentView
```

Comment:
Nit: It feels weird to me that DiscussionBoardFragmentView is doubly available within this module, via `DiscussionBoardFragmentView` or `discussion.views.DiscussionBoardFragmentView`.  Does this seem worth consolidating?

Reply:
Sure thing. I rebased and made that change.